app/assets/stylesheets/content/_forms.sass

```diff
@@ -552,3 +552,11 @@ fieldset.form--fieldset
 
   li, div
     padding: 0
+
+// change password form
+form.change_password
+  max-width: 31em
+  min-height: 31em
```

Comment:
@floriank  `min-height` I added to solve the problem with ` tooltip is cut off in FF under windows`. Bug defined in [this](https://community.openproject.org/work_packages/19752) ticket.

The `min-width` I use to make the form like in visuals provided [here](https://community.openproject.org/work_packages/18353). If briefly I just took what was done [here](https://github.com/finnlabs/openproject-zacero/blob/996bfc6b93d1f35e438a6542ab65c55af2e41061/app/assets/stylesheets/zacero/password.css.scss#L1-L77) and moved to core.